etc/ci-gather-coverage.sh

```diff
@@ -74,8 +74,21 @@ else
 fi;
 GAPInput
 
+# TODO/FIXME: for now we only support coveralls on Travis, but not e.g. on AppVeyor
+# (mainly because the coveralls support in the profiling package hardcodes Travis
+# into the JSON it generates)
+if [[ -n "$TRAVIS_JOB_ID" ]]
```

Comment:
So, is this still valid? I guess (hope?) not, then we could instead check here for the existence of `gap-coveralls.json`